src/formatting_stack/linters/eastwood.clj

```diff
@@ -17,15 +18,25 @@
     (-> eastwood.lint/default-opts
         (assoc :linters linters))))
 
+(def parallelize-linters? (System/getProperty "formatting-stack.eastwood.parallelize-linters"))
+
+(def config-filename "formatting_stack.clj")
+
+(assert (io/resource (str (io/file "eastwood" "config" config-filename)))
+        "The formatting-stack config file must exist and be prefixed by `eastwood/config`
+(note that this prefix must not be passed to Eastwood itself).")
+
 (defn lint! [{:keys [options]} filenames]
   (let [namespaces (->> filenames
                         (remove #(str/ends-with? % ".edn"))
                         (keep ns-name-from-filename))
         reports    (atom nil)
         output     (with-out-str
                      (binding [*warn-on-reflection* true]
-                       (eastwood.lint/eastwood (assoc options :namespaces namespaces)
-                                               (impl/->TrackingReporter reports))))]
+                       (cond-> options
+                         true                 (assoc :namespaces namespaces)
+                         parallelize-linters? (update :builtin-config-files conj config-filename)
```

Comment:
What adventage does `buildin-config-files` have over manually calling `set-linter-executor!`?

Reply:
It's Eastwood's intended API. Also, since it's classpath-based one can override `formatting_stack.clj` with e.g. a quick experiment.

I could remove the ns prefix here:

![image](https://user-images.githubusercontent.com/1162994/76704606-6a448300-66da-11ea-8a44-c6ebc896c12a.png)

to emphasize that this file is evaled with `*ns*` bound to `eastwood.util` (per its API)